ui/components/app/app-header/index.scss

```
@@ -107,4 +92,45 @@
       color: var(--color-primary-inverse);
     }
   }
+
+  &__beta-header {
+    background: var(--brand-colors-orange-orange400);
+    padding: 7px 10px;
```

General rule of thumb use values divisible by 8
```suggestion
    padding: 8px 8px;
```